### PR TITLE
Updated navbar with user's avatar

### PR DESCRIPTION
**Pull request by ritik-vashist**
updated #350 

Screenshot from 2021-06-07 16-22-17

@Abhishek-kumar09 , @vasudevsinghal  sir, please review my work.

### CODE REVIEW

File: src/components/NavBar/index.js [hunk 1]
```diff
@@ -102,18 +104,26 @@ function NavBar() {
...
+                </Typography>
+              )}
+
             </Hidden>
           </div>
           <Avatar
             className={classes.avatar}
-            src="https://media-exp1.licdn.com/dms/image/C560BAQHSXxzxdL45FA/company-logo_200_200/0/1588676153700?e=2159024400&v=beta&t=PEiMBARAmeCUpRrAuJZWev-F_oCZac_OunmUXcLuY5U"
+            src={`${user.photoURL}`}
           />
```
Comment (Abhishek-kumar09) [inline src/components/NavBar/index.js:127]: Render profile picture on if user is signed in.
```jsx
  {((user !== undefined) && (user !== null)) ? (
        <Avatar  
           className=
           src=
       /> ): ""
```

**Comment by ritik-vashist** [comment]
@Abhishek-kumar09  sir, i  updated the suggested changes kindly review my work.

**Review by Abhishek-kumar09 — COMMENTED** [review]
run `npm run lint:fix` from the terminal to fix codestyles

**Comment by ritik-vashist** [comment]
WhatsApp Image 2021-06-14 at 2 50 43 PM
@Abhishek-kumar09  Sir, no effect by this command what to do??

File: src/components/NavBar/index.js [hunk 2]
```diff
@@ -103,18 +105,26 @@ function NavBar() {
 
           <div className={classes.sectionDesktop}>
             <Hidden mdDown>
-              <Button className={classes.signInButton}>
-                <Typography noWrap>Sign In</Typography>
-              </Button>
-              <Button className={classes.signUpButton}>
-                <Typography noWrap>Sign Up</Typography>
-              </Button>
+              {user === undefined && user === null ? (
```
Comment (Abhishek-kumar09) [inline src/components/NavBar/index.js:108]:
```diff
-              {user === undefined && user === null ? (
+              {!user ? (
```

File: src/components/NavBar/index.js [hunk 3]
```diff
@@ -103,18 +105,26 @@ function NavBar() {
...
+              )}
             </Hidden>
           </div>
-          <Avatar
-            className={classes.avatar}
-            src="https://media-exp1.licdn.com/dms/image/C560BAQHSXxzxdL45FA/company-logo_200_200/0/1588676153700?e=2159024400&v=beta&t=PEiMBARAmeCUpRrAuJZWev-F_oCZac_OunmUXcLuY5U"
-          />
+          {
+            /* eslint-disable-next-line no-extra-boolean-cast */
+            !!user ? <Avatar className={classes.avatar} src={`${user.photoURL}`} /> : null
```
Comment (Abhishek-kumar09) [inline src/components/NavBar/index.js:126]:
```diff
-            !!user ? <Avatar className={classes.avatar} src={`${user.photoURL}`} /> : null
+            !!user ? <Avatar className={classes.avatar} src={`${user.photoURL}`} /> : ''
```

**Comment by ritik-vashist** [comment]
@Abhishek-kumar09  i have updated the suggested changes, kindly review it.

**Review by Abhishek-kumar09 — APPROVED** [review]

**Comment by Abhishek-kumar09** [comment]
image
Please correct

**Comment by ritik-vashist** [comment]
@Abhishek-kumar09  sir, i updated the file code. Please review it.

**Pull request merged**